metadata_tools/hosts/GO_0xxx/GO_0xxx_index_cloud.py

```
@@ -0,0 +1,82 @@
+import asyncio
+import os, sys
+from cloud_tasks.worker import Worker, WorkerData
+
+import metadata_tools.util as util
+import metadata_tools.common as com
+import host_config as hconf
+import index_config as config
+from metadata_tools.index_support import process_index, get_args
+
+#========================================================================================
+def process_task(task_id: str,
```

Ruff (0.14.10)

Unused function argument: `task_id`

(ARG001)